### PR TITLE
Add support for ppc64le

### LINKED ISSUE #2277: Add native support for the ppc64le architecture
POWER is the  processor architecture used for supercomputers and enterprise infrastructure deployments, specially for private cloud. As we already have Kubernetes running on Power, is is natural that the necessity for a tool that allows backing up data from cluster's storage emerges and Restic looks like the correct tool for this one (specially when combined with Velero). 

This issue is a feature request on adding official support for ppc64le.

### PR DESCRIPTION
**Pull request by rpsene**
This commit adds support for cross-compiling Restic for ppc64le
as part of the release process.

Add a new file:   changelog/unreleased/issue-2277
Modifies:   helpers/build-release-binaries/main.go
Modifies:   run_integration_tests.go

What is the purpose of this change? What does it change?
--------------------------------------------------------

Was the change discussed in an issue or in the forum before?
------------------------------------------------------------

closes #2277

Checklist
---------

- [x] I have read the [Contribution Guidelines](https://github.com/restic/restic/blob/master/CONTRIBUTING.md#providing-patches)
- [x] I have added documentation for the changes (in the manual)
- [x] There's a new file in `changelog/unreleased/` that describes the changes for our users (template [here](https://github.com/restic/restic/blob/master/changelog/TEMPLATE))
- [x] I have run `gofmt` on the code in all commits
- [x] All commit messages are formatted in the same style as [the other commits in the repo](https://github.com/restic/restic/blob/master/CONTRIBUTING.md#git-commits)
- [x] I'm done, this Pull Request is ready for review

### CODE REVIEW

**Comment by rpsene** [comment]
@fd0 ping - is there anything else required for this patch?

**Comment by Prajyot-Parab** [comment]
@fd0 are there any concerns that should be addressed here or we are good to merge this one?

**Comment by clnperez** [comment]
@skriss -- this is the restic pr, if you haven't seen it, that we'd love to get merged so we don't have to rely on builds from Unicamp for https://github.com/heptio/velero/pull/1768.

**Comment by rpsene** [comment]
@moritzdietz could you help us on this one?

**Comment by moritzdietz** [comment]
So I am not a maintainer so I can't speak for what goes into the default builds for each release. Nor can I commit to this repository. :)
The PR looks good and I have checked the CI jobs and didn't find anything weird about them.

We probably have to wait for the maintainers to come around here and give this a look.
On one side I don't see any problem with having another build for an architecture, on the other hand I don't know whether this is one which is broadly used by folks.
(Just a side note: this PR would add ppc64le support here, but not for the restic beta builds. Those are in a different repository :) )

**Comment by rpsene** [comment]
@moritzdietz thanks for the reply. What is this repo for beta builds? btw, ppc64le is used by the #1 and #2 fastest supercomputers available today and for lots of big companies :)

**Comment by moritzdietz** [comment]
The repository for the beta build is at https://github.com/restic/beta

**Comment by rpsene** [comment]
Thanks again @moritzdietz. Given that this repo contains only 2 files and no info about a CICD mechanism, do you know where it is built?

**Comment by rpsene** [comment]
@moritzdietz @fd0 I have raised a PR in the beta repo https://github.com/restic/beta/pull/1

**Comment by rpsene** [comment]
@nrb do you have bandwidth to take a look at this one?

**Comment by rpsene** [comment]
@fw42 do you have bandwidth to take a look at this one?

**Review by mholt — APPROVED** [review]
I don't know much about the build process myself, but I know we compile caddy for this architecture and it works fine, so, I will approve this.

**Comment by rpsene** [comment]
@mholt thanks! Are you a good candidate to take a look at https://github.com/restic/beta/pull/1 ?

**Comment by carlisia** [comment]
Does this PR need more approvals? Else, what is pending for it to be merged? Thank you!

File: changelog/unreleased/issue-2277
Comment (rawtaz) [inline changelog/unreleased/issue-2277]: s/add/Add/

File: changelog/unreleased/issue-2277
Comment (rawtaz) [inline changelog/unreleased/issue-2277]: There's no need to mention Restic here, since this is already a natural context. Also, the grammar needs fixing.

File: changelog/unreleased/issue-2277
Comment (rawtaz) [inline changelog/unreleased/issue-2277]: The grammar of this text needs to be fixed. Also, it doesn't need all that much background about what POWER is about. And what is Velero anyway? In other words, please make the text more relevant for a changeling entry :)

**Comment by rawtaz** [comment]
@carlisia That the project maintainer gives thumbs up or down, because it's a change that he must feel comfortable with to maintain in the future. It's easy to add changes, but someone has to maintain them.

I checked in the referenced issue but couldn't find any discussion about this change where @fd0 has commented, so even though @mholt and @moritzdietz commented that they think it looks good, it's probably a good idea to make sure @fd0 is on board.

**Comment by carlisia** [comment]
Thanks @rawtaz. I appreciate the explanation and your review.

**Comment by rpsene** [comment]
Good the see we have activity on this PR, I will send the fixes suggested. @carlisia @rawtaz I hope the thumbs up requirement from @fd0 does not delay it again.

File: changelog/unreleased/issue-2277
Comment (rawtaz) [inline changelog/unreleased/issue-2277]: I'm sorry if you feel I'm nitpicking, but this text shouldn't be indented :)

**Comment by rawtaz** [comment]
I'm pretty sure he'll check it soon :)

File: changelog/unreleased/issue-2277
Comment (rpsene) [reply changelog/unreleased/issue-2277]: done!

**Review by fd0 — APPROVED** [review]
LGTM, thank you very much (and I'm really sorry for the long time you had to wait for this one).

**Comment by carlisia** [comment]
Can this be merged then? :)

**Comment by rpsene** [comment]
> Can this be merged then? :)

who will have the honor?

**Comment by mholt** [comment]
Dibs

**Comment by rawtaz** [comment]
@carlisia Please note and understand that repeated "reminders" with this frequency is not constructive, it's causing nothing but annoyance. Things don't get merged quicker because you ask about it multiple times. We *were* already working on it.

**Pull request merged**
